codestyle.xml

```diff
@@ -0,0 +1,23 @@
+<code_scheme name="spinnaker">
```

Comment:
I'm not an expert on intellij, so I'm mainly looking to confirm that if somebody follows the 2 steps in 'Apply groovy code formatting scheme', that no reformatting immediately takes place.